### PR TITLE
[CL-351] Prevent tw-group styling conflicts by namespacing

### PR DESCRIPTION
**Pull request by vleague2**
## 🎟️ Tracking

[CL-351](https://bitwarden.atlassian.net/browse/CL-351)

## 📔 Objective

In PR #10188, we fixed an issue where styles were colliding due to multiple `tw-group`s being on a page together. This was solved by namespacing the `tw-group` for that instance. 

This PR namespaces the remaining `tw-group`s used in the app to prevent the issue from occurring elsewhere. Some of the existing `tw-group`s were actually not being used (leftovers from previous implementations of components), so those were removed.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[CL-351]: https://bitwarden.atlassian.net/browse/CL-351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/components/src/table/sortable.component.ts [hunk 1]
```diff
@@ -97,7 +97,6 @@ export class SortableComponent implements OnInit {
 
   get classList() {
     return [
-      "tw-group",
```
Comment (vleague2) [inline libs/components/src/table/sortable.component.ts:100]: ℹ️ Usage removed in [this PR](https://github.com/bitwarden/clients/pull/4676/files#diff-57c3393427521c7d00e707c95def780e1a131558576e05df63525a0b62deba66L79)

File: libs/components/src/layout/layout.component.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
     >
   </nav>
 </div>
-<div class="tw-group tw-flex tw-w-full">
```
Comment (vleague2) [inline libs/components/src/layout/layout.component.html:16]: ℹ️ Introduced in [this PR](https://github.com/bitwarden/clients/pull/6383/files#diff-07d89780a710f18d7242e4abbc95f91b6fd23c725dcb34b11a0568462f61417bR16) but never used (likely from a previous commit in that PR)

File: apps/web/src/app/settings/domain-rules.component.html [hunk 3]
```diff
@@ -54,7 +54,7 @@ <h2 bitTypography="h2" class="spaced-header">{{ "globalEqDomains" | i18n }}</h2>
     </p>
     <bit-table *ngIf="!loading && global.length > 0">
       <ng-template body>
-        <tr bitRow *ngFor="let d of global" class="tw-group">
```
Comment (vleague2) [inline apps/web/src/app/settings/domain-rules.component.html:57]: ℹ️ Introduced in [this PR](https://github.com/bitwarden/clients/pull/8745/files#diff-d4c4e71db8ebd58d8b23df14dd5c7a7a31f131fcdaaf5724da8f892adf34807eR57), never used and not present pre-migration to CL

File: apps/web/src/app/billing/organizations/change-plan-dialog.component.ts [hunk 4]
```diff
@@ -348,7 +348,6 @@ export class ChangePlanDialogComponent implements OnInit, OnDestroy {
     switch (cardState) {
       case PlanCardState.Selected: {
         return [
-          "tw-group",
```
Comment (vleague2) [inline apps/web/src/app/billing/organizations/change-plan-dialog.component.ts:351]: ℹ️ Usage removed in [this PR](https://github.com/bitwarden/clients/pull/10662/files#diff-87999f1788607ef01a7f84fe24d53d71bb51214412351184ec76f23ae3697b5cL57)

**Review by willmartian — APPROVED** [review]
Looks great! FYI, Vicki also looked into adding an eslint rule to error on this, but a number of our eslint rules are actually not working right now. Hopefully that can be a future addition.

**Review by cturnbull-bitwarden — APPROVED** [review]

**Pull request merged**